object/verification.go

```diff
@@ -26,6 +26,10 @@ import (
 	"xorm.io/core"
 )
 
+const (
+	wrongCode = "wrongCode"
```

Comment:
Herer is `wrongCode `, it is diferent from the result that function `CheckVerificationCode(dest, code, lang)` return in line 199